SemiTruck.h

```diff
@@ -1,2 +1,15 @@
 #pragma once
 
+#include "Vehicle.h"
+
+struct SemiTruck : Vehicle
+{
+    SemiTruck(const std::string& t);
+
+    virtual ~SemiTruck() override;
+    SemiTruck(const SemiTruck&);
+    SemiTruck& operator=(const SemiTruck&) = default;
```

Comment:
move this copy-assign operator implementation to the cpp file, with the other implementations.
- [ ] revise